api/swagger.yaml

```diff
@@ -665,7 +665,17 @@ definitions:
               type: "string"
           IpcMode:
             type: "string"
-            description: "IPC namespace to use for the container."
+            description: |
+                    IPC sharing mode for the container. Possible values are:
+
+                    - `"none"`: own private IPC namespace, with /dev/shm not mounted
+                    - `"private"`: own private IPC namespace
+                    - `"shareable"`: own private IPC namespace, with a possibility to share it with other containers
+                    - `"container:<name|id>"`: join another (shareable) container's IPC namespace
+                    - `"host"`: use the host system's IPC namespace
```

Comment:
For a follow-up PR we can also add an `enum` in the swagger specs; https://github.com/moby/moby/blob/1a0fbc4a6facd35ca93e4eded902e73d0d4b8378/api/swagger.yaml#L171-L174